src/inference_result.cc

```diff
@@ -142,7 +144,8 @@ void InferenceResult::dump_weights_in_binary(std::ofstream &binary_output) const
 }
 
 // FIXME: See note on load_assignments.
-void InferenceResult::dump_assignments_in_binary(std::ofstream &binary_output) const {
+void InferenceResult::dump_assignments_in_binary(
+    std::ofstream &binary_output) const {
```

Comment:
In general I think `ostream` is more reusable than `ofstream`.  Not sure what you can do more with the `f`.